tests/app/api/services/test_abr.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+
+from app.api.services import abr_service
+import requests
+import unittest
+import mock
+from mock import patch
+
+
+class TestAbrService(unittest.TestCase):
```

Comment:
I would stick with pytest since other tests use that.  Is there a reason why you used `unittest`?